core-web/libs/data-access/src/lib/dot-properties/dot-properties.service.ts

```diff
@@ -52,4 +62,23 @@ export class DotPropertiesService {
             .get('/api/v1/configuration/config', { params: { keys: `list:${key}` } })
             .pipe(take(1), pluck('entity', key));
     }
+
+    /**
+     * Loads the configuration for the feature flags by calling the `getKeys` method and subscribing to the result.
+     * @returns An observable that emits the feature configuration object.
+     */
+    loadConfig() {
+        this.getKeys([
+            FeaturedFlags.LOAD_FRONTEND_EXPERIMENTS,
+            FeaturedFlags.DOTFAVORITEPAGE_FEATURE_ENABLE,
+            FeaturedFlags.FEATURE_FLAG_TEMPLATE_BUILDER,
+            FeaturedFlags.FEATURE_FLAG_SEO_IMPROVEMENTS,
+            FeaturedFlags.FEATURE_FLAG_SEO_PAGE_TOOLS,
+            FeaturedFlags.FEATURE_FLAG_EDIT_URL_CONTENT_MAP
```

Comment:
Can we do `Object.values()` to get the features flags?

Reply:
Not exactly, FeaturedFlags is a enum, buti add a const array with all enabled feature flags, and import directly here

Reply:
Enums in TypeScript are represented as objects in JavaScript @KevinDavilaDotCMS.

You should be able to do

```ts
const FEATURE_FLAG_VALUES = Object.values(FeaturedFlags); // This is a dynamic way to get the values of the feature flags
```